tests/test_loader.py

```diff
@@ -42,6 +48,11 @@ def test_filesystem_loader(self, filesystem_loader):
         assert tmpl.render().strip() == 'BAR'
         tmpl = env.get_template('foo/test.html')
         assert tmpl.render().strip() == 'FOO'
+        if has_pathlib:
```

Comment:
Write a separate test rather than adding to an existing test. Use `pytest.mark.skipif` to skip the test if pathlib isn't available.